test/test_torch.py

```diff
@@ -5190,6 +5190,27 @@ def test_dot_data_use(self):
             model.weight.data = weight
             out = model(input)
 
+    def test_cpp_stacktraces(self):
+        python_error_msg = "Trying to create tensor with negative dimension"
+        cpp_stack_msg = "Exception raised from check_size_nonnegative at"
+
+        def check_error(has_cpp_stack_trace):
+            raised = False
+            try:
+                torch.rand(-1)
+            except Exception as e:
+                raised = True
```

Comment:
You don't seem to do anything with this variable

Reply:
Good catch, this is missing an assert at the end of the function.